setup.cfg

```diff
@@ -23,19 +24,16 @@ classifiers =
 [options]
 include_package_data = True
 packages = find:
-python_requires = >=3.7
+python_requires = >=3.12
 zip_safe = False
 install_requires =
     blinker>=1.4
     Flask>=3.0.0
     itsdangerous>=2.1.0
-    MarkupSafe>=0.23
+    MarkupSafe>=2.1.0
```

Comment:
please don't sneak in changes which are not part of the type hints change!

Reply:
yes, my vscode suggested, then i saw that is introduced better type hinting support. But i can revert.